js/front/dataset/resource-modal.vue

```diff
@@ -1,36 +1,36 @@
 <template>
 <modal class="resource-modal" v-ref:modal
-    :title="resource.name">
+    :title="resource.title">
 
     <div class="modal-body">
-        {{{ resource.description }}}
+        {{{ resource.description|markdown }}}
 
         <dl class="dl-horizontal dl-wide">
           <dt>{{ _('URL') }}</dt>
-          <dd><a :href="resource.contentUrl" @click="onClick">{{resource.contentUrl}}</a></dd>
+          <dd><a :href="resourceJsonLd.contentUrl" @click="onClick">{{resourceJsonLd.contentUrl}}</a></dd>
```

Comment:
Why not user `resource` everywhere ? This will allow to strip from the JSON-LD every non standard values we added